OBAKit/Models/dao/OBAModelDAO.m

```diff
@@ -376,6 +376,14 @@ - (OBABookmarkGroup*)todayBookmarkGroup {
     }
 }
 
+- (NSInteger)todayBookmarkGroupCount {
+    if (self.todayBookmarkGroup) {
+        return [self.todayBookmarkGroup.bookmarks count];
```

Comment:
Being explicit is ok, but for future reference, Objective-C treats `nil` and `0` as being essentially interchangeable, so you could shorten this method to just:

    - (NSInteger)todayBookmarkGroupCount {
        return self.todayBookmarkGroup.bookmarks.count;
    }